After upgrade to through@2.3.10, build failure with error Cannot find moudle 'call-bind'

Hi ljhard,

When my project upgrade to 2.3.10, the build failure with the error Cannot find moudle 'call-bind'

Do you know hot to fix this error?


Thanks,
Charles

Hi @ljharb, it looks like the latest version of `through` has a missing dependency on `call-bind`. 

This prevents [`inquirer`](https://www.npmjs.com/package/inquirer) from running (see https://github.com/SBoudrias/Inquirer.js/issues/1317) and affects all the projects that rely on it, including WebdriverIO https://github.com/webdriverio/webdriverio/issues/11399:

```
[wdio] Error: Cannot find module 'call-bind'
[wdio] Require stack:
[wdio] - /Users/example/tmp/node_modules/@ljharb/through/index.js
[wdio]     at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
[wdio]     at Module._load (node:internal/modules/cjs/loader:922:27)
[wdio]     at Module.require (node:internal/modules/cjs/loader:1143:19)
[wdio]     at require (node:internal/modules/cjs/helpers:119:18)
[wdio]     at Object.<anonymous> (/Users/example/tmp/node_modules/@ljharb/through/index.js:4:16)
[wdio]     at Module._compile (node:internal/modules/cjs/loader:1256:14)
[wdio]     at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
[wdio]     at Module.load (node:internal/modules/cjs/loader:1119:32)
[wdio]     at Module._load (node:internal/modules/cjs/loader:960:12)
[wdio]     at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
[wdio]     at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
[wdio]   code: 'MODULE_NOT_FOUND',
```

I believe that PR https://github.com/ljharb/through/pull/2 and a new release of `through` should resolve this issue.

Thanks!

--- a/package.json
+++ b/package.json
@@ -15,6 +15,9 @@
 		"version": "auto-changelog && git add CHANGELOG.md",
 		"postversion": "auto-changelog && git add CHANGELOG.md && git commit --no-edit --amend && git tag -f \"v$(node -e \"console.log(require('./package.json').version)\")\""
 	},
+	"dependencies": {
+		"call-bind": "^1.0.2"
+	},
 	"devDependencies": {
 		"@ljharb/eslint-config": "^21.1.0",
 		"aud": "^2.0.3",

